pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/signature/OperationSignature.java

```diff
@@ -62,29 +72,15 @@ public static OperationSignature buildFor(ASTMethodOrConstructorDeclaration node
         return POOL.get(code);
     }
 
-    /** Used internally by the pooler. */
-    private static int code(Visibility visibility, Role role, boolean isAbstract) {
-        return visibility.hashCode() * 31 + role.hashCode() * 2 + (isAbstract ? 1 : 0);
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        return o instanceof OperationSignature && super.equals(o) && role == ((OperationSignature) o).role
-            && isAbstract == ((OperationSignature) o).isAbstract;
-    }
-
-    @Override
-    public int hashCode() {
-        return super.hashCode() * 2 + role.hashCode() * 4 + (isAbstract ? 1 : 0);
-    }
 
     /**
      * Role of an operation.
      */
     public enum Role {
         GETTER_OR_SETTER, CONSTRUCTOR, METHOD, STATIC;
 
-        private static final Pattern NAME_PATTERN = Pattern.compile("(?:get|set|is|increment|decrement)\\w*");
+
+        //   private static final Pattern GETTER_OR_SETTER_NAME_PATTERN = Pattern.compile("(?:get|set|is)\\w*");
```

Comment:
I'll merge this PR now. But could you please remove this line in a later commit, if it is not needed anymore? Thanks!